docs/get_started.rst

```diff
@@ -1,17 +1,348 @@
+    If that's completely new to you, we recommend starting with the `CumulusCI setup module <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup>`_ module on Trailhead to walk you through step by step.
+
+
+On macOS
+^^^^^^^^
+Ensure you have `Homebrew <https://brew.sh/>`_ installed as it is a prerequisite for installing CumulusCI on macOS.
+
+
+Install via ``pipx``
+******************************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Python software on your computer.
+First install ``pipx`` with the following two commands::
+
+    $ brew install pipx
+    $ pipx ensurepath
+
+
+Once ``pipx`` is installed, you can install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished you can `verify your installation`_.
+
+
+
+On Linux
+^^^^^^^^
+
+
+
+Install via ``pipx``
+**************************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Python software on your computer.
+Installation instruction for ``pipx`` can be found `here <https://pipxproject.github.io/pipx/installation/>`_.
+Once ``pipx`` is installed, we can install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished you can `verify your installation`_.
+
+
+
+On Windows
+^^^^^^^^^^
+
+
+
+Install Python 3
+********************
+#. Go to the `Python downloads page <https://www.python.org/downloads/release/python-383/>`_.
+#. Download the latest Python 3 release. Most users should select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it may depend on your particular computer setup.
+#. Use the installation wizard to install:
+
+   * Check the "Add Python <version> to PATH" checkbox
+   * Click "Install Now"
+
+.. image:: images/windows_python.png
+
+
+
+Install via ``pipx``
+***********************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Python software on your computer.
+Open your preferred terminal application
+(e.g. `CMD.exe <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd>`_ on Windows).
+If you already have your terminal open, close it and reopen it. Enter the following command::
+
+    $ python -m pip install --user pipx
+
+.. image:: images/pipx.png
+
+Next we want to modify the default PATH environment variable to include pipx::
+
+    $ python -m pipx ensurepath
+
+Open a new command prompt and verify that pipx is available::
+
+    $ pipx --version
+
+You should see a version number after entering in this command, such as: ``0.15.5.1``.
+
+You can now install CumulusCI with::
+
+    $ pipx install cumulusci
```

Comment:
running build_ext
    building 'lxml.etree' extension
    error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\Cohen\.local\pipx\venvs\cumulusci\Scripts\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Cohen\\AppData\\Local\\Temp\\pip-install-hm7pczuo\\lxml\\setup.py'"'"'; __file__='"'"'C:\\Users\\Cohen\\AppData\\Local\\Temp\\pip-install-hm7pczuo\\lxml\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Cohen\AppData\Local\Temp\pip-record-7l8kg0u0\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\Cohen\.local\pipx\venvs\cumulusci\include\site\python3.9\lxml' Check the logs for full command output.

Reply:
@acrosman Is this the issue you ran into?

Reply:
Yes.  After installing the CPP tool from that link we ran into errors related to libxml2 missing.  I can get a more detailed error tomorrow if it would be useful

Reply:
@acrosman You're definitely not supposed to have to install a C compiler to use CumulusCI on Windows! Is it possible you're trying to use Python 3.9 instead of 3.8? The maintainers of the lxml library have not yet published binary distributions of the package for Python 3.9, so pip falls back to trying to compile it from source. We don't support CumulusCI on Python 3.9 yet, largely for this reason.

Reply:
Not my computer so looking at context in his error messages yes 3.9.  I'll help him install 3.8.

Reply:
We did get this sorted out but it took some doing.  I don't know how it would be helpful to provide feedback, but once you get down the 3.9 it is quite a challenge to walk it back.  

Once we have Python 3.8 up and running pipx continued to run off the partially failed install, which contained links to 3.9.  We had to remove all the pipx installed shared scripts and libraries (so everything at %USERPROFILE%\.local\pipx had to be deleted manually).  Once we deleted that content, pipx happily loaded python 3.8 and we were off to the races.

As best I could determine pipx currently contains no built-in mechanism to recover from this state.